com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/Constants.java

```diff
@@ -168,4 +168,6 @@
     public static final String TOOL_TYPE = "tool_type";
     public static final String FEEDBACK_MAIL = "feedback_form";
 
+    protected Constants() {

```

Comment:
private?

Reply:
The constructor is protected because this class is overrided